pkg/apiserver/apiserver.go

```diff
@@ -56,6 +56,13 @@ var (
 		},
 		[]string{"verb", "resource", "client", "code"},
 	)
+	realTimeRequestCounter = prometheus.NewGaugeVec(
```

Comment:
It's a gauge, but I'm using it to count requests so I felt calling it a counter (in the literal sense) would be clearer